### PR TITLE
2022.04+lf 5.15.52 2.1.0-fio initial support of 7ulp-evk

### PR DESCRIPTION
**Pull request by MrCry0**
Incomplete, but it allows us to build imx7ulp-evk target.

### CODE REVIEW

File: board/freescale/mx7ulp_evk/mx7ulp_evk.c [hunk 1]
```diff
@@ -274,3 +300,4 @@ int show_bootloader_menu(void) {
 }
 #endif /* CONFIG_BOOTLOADER_MENU */
 #endif /* CONFIG_ANDROID_SUPPORT*/
+
```
Comment (ricardosalveti) [inline board/freescale/mx7ulp_evk/mx7ulp_evk.c]: nit, no need for this empty line

**Comment by ricardosalveti** [comment]
Thanks for working on this.

File: board/freescale/mx7ulp_evk/mx7ulp_evk.c (hunk 1 above)
Comment (MrCry0) [reply board/freescale/mx7ulp_evk/mx7ulp_evk.c]: @ricardosalveti thanks! Fixed.

**Comment by MrCry0** [comment]
Closed in favor of https://github.com/foundriesio/u-boot/pull/200

**Comment by quaresmajose** [comment]
> Closed in favor of #195

195 is the same, maybe is https://github.com/foundriesio/u-boot/pull/200

**Pull request closed without merging**